singer_sdk/target_base.py

```diff
@@ -571,4 +576,46 @@ def cli(
 class SQLTarget(Target):
     """Target implementation for SQL destinations."""
 
+    @classproperty
+    def capabilities(self) -> list[CapabilitiesEnum]:
+        """Get target capabilities.
+
+        Returns:
+            A list of capabilities supported by this target.
+        """
+        sql_target_capabilities: list[CapabilitiesEnum] = super().capabilities
+        sql_target_capabilities.extend([TargetCapabilities.TARGET_SCHEMA])
+
+        return sql_target_capabilities
+
+    @classmethod
+    def append_builtin_config(self, config_jsonschema: dict) -> None:  # noqa: ANN102
```

Comment:
@edgarrmondragon I don't know how to fix this and was wondering if you could weigh in.  If I remove the `@classmethod` then `meltano invoke target-mssql --about` fails but when I leave it in `pre-commit` checks fail asking me to specify a class type for cls.  That is why I have `# noqa: ANN102`

Reply:
@BuzzCutNorman the `cls` parameter needs a class annotation, i.e.

```python
@classmethod
def append_builtin_config(cls: type[SQLTarget], config_jsonschema: dict) -> None:
```

I've pushed this change because I wasn't sure myself if that was the sole issue.

In the future, we should probably just ignore `ANN102` in the whole repo.

Reply:
@edgarrmondragon ahh got it.  Thanks 🙏 for that.  `target-mssql --about` tested good post commit 🎉